### PR TITLE
Update gh actions blog

### PR DESCRIPTION
**Pull request by schustmi**
## Pre-requisites
Please ensure you have done the following:
- [x] I have read the **CONTRIBUTING.md** document.
- [x] If my change requires a change to the documentation, I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Describe changes
Briefly describe the changes you have introduced.

### CODE REVIEW

File: _posts/2022-06-20-github-actions-orchestrator.md [hunk 1]
```diff
@@ -238,17 +181,31 @@ Now that we're done setting up and configuring all our infrastructure and extern
...
 ```
+
+### Deploying ZenML
+
+To use ZenML with remote orchestrators like the GitHub Actions orchestrator, you need to deploy ZenML.
+Follow [this guide](https://docs.zenml.io/getting-started/deploying-zenml#deploying-zenml-in-the-cloud-remote-deployment-of-the-http-server-and-database) to
+do so.
+
+Once the deployment is finished, let's connect to it by running the following command and logging in with
+the username and passord you set during the deployment phase:
```
Comment (dnth) [inline _posts/2022-06-20-github-actions-orchestrator.md:200]:
```suggestion
the username and password you set during the deployment phase:
```

**Review by dnth — APPROVED** [review]
Looks good to me other than a minor typo.

File: _posts/2022-06-20-github-actions-orchestrator.md [hunk 2]
```diff
@@ -292,38 +240,22 @@ A [ZenML stack](https://docs.zenml.io/advanced-guide/stacks-components-flavors)
...
-zenml secrets-manager secret register azure_mysql_auth \
-    --schema=mysql \
-    --user=<MYSQL_USERNAME> \
-    --password=<MYSQL_PASSWORD> \
-    --ssl_ca=@<PATH_TO_SSL_CERTIFICATE>
-```
-
-For the artifact store secret, we're going to need the **storage account name** and **key** that we saved when we [created our storage account earlier](#create-a-storage-account).
+Once the stack is active, we can register the secret that ZenML needs to authenticate with our artifact store:
+We're going to need the **storage account name** and **key** that we saved when we [created our storage account earlier](#create-a-storage-account).
```
Comment (strickvl) [inline _posts/2022-06-20-github-actions-orchestrator.md:258]:
```suggestion
Once the stack is active, we can register the secret that ZenML needs to authenticate with our artifact store.
We're going to need the **storage account name** and **key** that we saved when we [created our storage account earlier](#create-a-storage-account):
```

**Review by strickvl — APPROVED** [review]
Likewise looks good aside from one tiny suggestion.

**Pull request merged**